File: tests/corpus/negative-fixtures/QA-CS-107/FeedTests.cs
using Microsoft.Playwright;
using Microsoft.Playwright.MSTest;
using Microsoft.VisualStudio.TestTools.UnitTesting;

[TestClass]
public class FeedTests
{
    [TestMethod]
    public async Task LoadsFeed()
    {
        await Page.GotoAsync("https://example.com/feed");
        await Page.WaitForLoadStateAsync(LoadState.DOMContentLoaded);
        Assert.IsNotNull(Page);
    }

    [TestMethod]
    public async Task WaitsForResponse()
    {
        await Page.WaitForResponseAsync("*/api/feed*");
        Assert.IsNotNull(Page);
    }
}

File: tests/corpus/negative-fixtures/QA-JV-107/FeedTests.java
import com.microsoft.playwright.*;
import com.microsoft.playwright.options.LoadState;
import org.junit.jupiter.api.Test;

public class FeedTests {

    @Test
    public void loadsFeed(Page page) {
        page.navigate("https://example.com/feed");
        page.waitForLoadState(LoadState.DOMCONTENTLOADED);
    }

    @Test
    public void waitsForResponse(Page page) {
        page.waitForResponse(url -> url.contains("/api/feed"));
    }
}
